python/tvm/relay/ir_pass.py

```diff
@@ -652,3 +652,19 @@ def partial_evaluate(expr):
       The output expression.
     """
     return _ir_pass.partial_evaluate(expr)
+
+def canonicalize_expr(expr):
```

Comment:
as we move toward the new pass API, let us simply remove the expr->expr API here

Reply:
@tqchen @zhiics how should I update unittest to use mod->mod API?

Reply:
I think you can delete this function here, but just keep the one in transform.py

To use mod->mod API, you can do something like this in the unit test:
```python
mod = _module.Module.from_expr(func)
seq = _transform.Sequential([transform.CanonicalizeExpr(), transform.InferType()])
with _transform.PassContext(opt_level=3):
    mod = seq(mod)
canonicalized_expr = mod[mod.entry_func.name_hint]
```